pkg/util/log/log.go

```diff
@@ -394,6 +412,20 @@ func Error(v ...interface{}) error {
 	return err
 }
 
+// ErrorStackDepth logs at the error level and the given stack depth and returns an error containing the formated log message
```

Comment:
nit: it'd be nice to add in the comment here in which cases it's useful to use an additional stack depth